plugins/management-console/src/main/java/ai/djl/serving/console/ConsoleRequestHandler.java

```diff
@@ -425,19 +427,31 @@ private String getLastLineText(File file, int lines) {
                         break;
                     }
                 }
-                builder.append(c);
+                bos.write(c);
                 fileLength = fileLength - pointer;
             }
-            builder.reverse();
         } catch (IOException e) {
             throw new InternalServerException("Failed to read log file.", e);
         }
-        return builder.toString();
+        return new String(reverse(bos.toByteArray()), StandardCharsets.UTF_8);
```

Comment:
I'm definitely missing something here, but why does`bos.toString(StandardCharsets.UTF_8)` not work for us here? I'm guessing it's the reversal of the array, but i'm also confused why that's needed.

Reply:
oh i see, i exanded the above code and we're reading in reverse. makes sense.